tools/harness-thci/README.md

```diff
@@ -28,3 +29,28 @@ new THCI implementation.
 6. Drag OpenThread: TI CC2538DK reference device to Test Bed list with desired number.
 
 7. Select one or multiple test cases to execute.
+
+
+## OpenThread WpanCtl Environment Setup ##
+
+1. Copy the "OpenThread.png" to \GRL\Thread1.1\Web\images folder.
+
+2. Copy the "deviceInputFields.xml" to \GRL\Thread1.1\Web\data folder.
+
+3. Copy the "OpenThread_WpanCtl.py" to \GRL\Thread1.1\Thread_Harness\THCI folder.
+
+4. Connect the NCP board (nRF52840) to Raspberry Pi's usb port and make sure wpanctl command work.
+
+5. Connect the Raspberry Pi's GPIOs (For Raspberry Pi 3B, link GPIO14 as TXD, GPIO15 as RXD and Pin14 as GND) with
+   a UART2USB adapter.
+
+6. Connect DUT(with Step 5's adapter), sniffer and other golden devices as reference device to Host PC.
+
+7. Get the DUT serial port hardware identifier and add a new platform group named OpenThread_WpanCtl in
+   \GRL\Thread1.1\Config\Configuration.ini referring to https://openthread.io/certification/automation-setup#acting_as_a_new_reference_platform.
```

Comment:
"...named OpenThread_WpanCtl in \GRL\Thread1.1\Config\Configuration.ini.  See https://openthread.io/certification/automation-setup#acting_as_a_new_reference_platform for more information."

Reply:
@FiveDimensions Please update this.